src/routes/api/profile/education/academic-training/update/+server.ts

```diff
@@ -0,0 +1,28 @@
+import { BACKEND_BASE_URL } from '$env/static/private'
+import { json } from '@sveltejs/kit'
+
+export interface AcademicTrainingCreationPayload {
+  studyId: number
+  name: string
+  universityName: string
+  degree: string
+  graduationYear: string
+}
+
+export async function POST({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```

Comment:
```suggestion
export async function PUT({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```
Como actualiza algo existente, debiera ser un PUT en vez de un POST. Y sí, SvelteKit sí lo soporta https://learn.svelte.dev/tutorial/other-handlers